src/lolex-src.js

```diff
@@ -561,6 +574,32 @@ function withGlobal(_global) {
             return [secsSinceStart, remainderInNanos];
         }
 
+        clock.requestIdleCallback = function requestIdleCallback(func, timeout) {
+            var timeToNextIdlePeriod = 0;
+
+            if (clock.countTimers() > 0) {
+                timeToNextIdlePeriod = 50; // const for now
```

Comment:
This is a bit problematic since if there is an idle callback already queued we will sequence the idle callbacks incorrectly.

For example:

 - add timer for after 50ms
 - add idle callback
 - tick 20ms
 - add idle callback

The timer will run between the idle callbacks.

I'm fine with merging and adding this as a todo though.